libse/SubtitleFormats/SonyDVDArchitect.cs

```diff
@@ -66,30 +66,18 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
                 if (line.Length > 26 && line[2] == ':')
                     match = Regex.Match(line);
 
-                if (match != null && match.Success)
+                if (match?.Success == true)
                 {
                     string s = line.Substring(0, match.Length);
                     s = s.Replace(" - ", ":");
                     s = s.Replace(" ", string.Empty);
                     string[] parts = s.Split(':');
                     if (parts.Length == 8)
                     {
-                        int hours = int.Parse(parts[0]);
-                        int minutes = int.Parse(parts[1]);
-                        int seconds = int.Parse(parts[2]);
-                        int milliseconds = int.Parse(parts[3]) * 10;
-                        var start = new TimeCode(hours, minutes, seconds, milliseconds);
-
-                        hours = int.Parse(parts[4]);
-                        minutes = int.Parse(parts[5]);
-                        seconds = int.Parse(parts[6]);
-                        milliseconds = int.Parse(parts[7]) * 10;
-                        var end = new TimeCode(hours, minutes, seconds, milliseconds);
-
                         string text = line.Substring(match.Length).TrimStart();
+                        text = text.Replace("\r", Environment.NewLine);

```

Comment:
The default line separator delimiter used when serializing text if `\r`  https://github.com/SubtitleEdit/subtitleedit/pull/1893/files#diff-f2f1a9c4446a62a5d7b7dd5135546ae5R41.